tests/test_abc.py

```diff
@@ -10,6 +11,7 @@ class Model(BaseModel, abc.ABC):
         some_field: str
 
 
+@pytest.mark.skipif(sys.version_info < (3, 12), reason='error value different on older versions')
```

Comment:
I'd rather don't throw out tests for everything except an unreleased Python version.

Reply:
As discussed, we are fine with this change.